packages/twenty-e2e-testing/tests/login.setup.ts

```diff
@@ -1,5 +1,7 @@
 import { test as setup, expect } from '@playwright/test';
-import path from 'path';
+import { config } from 'dotenv';
+import path = require('path');
+config({ path: path.resolve(__dirname, '..', '.env') });
```

Comment:
I'm unsure if we want to rely on an environment variable in an E2E test, and I think we don't want them to be parameterizable.

I would create "mock" data for e2e tests like this:

```ts
const CURRENT_USER = {
  email: 'xx@xx.com',
  password: '1234',
}
```

Relying on environment variables is unsafe. What happens if the `DEFAULT_LOGIN` is set as an empty string or is not defined?

Reply:
If `DEFAULT_LOGIN` is not defined or empty, then Playwright will throw an error `Error: locator.fill: value: expected string, got undefined at .../login.setup.ts:9:40`

Reply:
I deleted this, the reason why I added these lines was thanks to wrong setup on my side - working directory changed to root folder of project where it should be a root folder of E2E testing package